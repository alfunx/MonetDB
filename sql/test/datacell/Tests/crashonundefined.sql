select * from [select * from undefinedbasket] as t;
