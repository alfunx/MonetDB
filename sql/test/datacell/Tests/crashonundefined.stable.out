stdout of test 'crashonundefined` in directory 'sql/test/datacell` itself:


# 12:41:35 >  
# 12:41:35 >   mserver5 "--config=/ufs/niels/scratch/MonetDB/Linux-x86_64/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=4 --set "monet_mod_path=/ufs/niels/scratch/MonetDB/Linux-x86_64/lib/MonetDB5:/ufs/niels/scratch/MonetDB/Linux-x86_64/lib/MonetDB5/lib:/ufs/niels/scratch/MonetDB/Linux-x86_64/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/niels/scratch/MonetDB/Linux-x86_64/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=35517 --set xrpc_port=48343 --set monet_prompt= --set mal_listing=2 --trace  "--dbname=mTests_src_test_datacell" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 12:41:35 >  

# MonetDB server v5.21.0, based on kernel v1.39.0
# Serving database 'mTests_src_test_datacell', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.754 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://alf.ins.cwi.nl:35517/
# MonetDB/SQL module v2.39.0 loaded
# MonetDB/SQL module v2.39.0 loaded

Ready.


# 12:41:35 >  
# 12:41:35 >  mclient -lsql -ftest -i -e --host=alf --port=35517 
# 12:41:35 >  


# 12:41:35 >  
# 12:41:35 >  Done.
# 12:41:35 >  

